contracts/GnosisSafe.sol

```diff
@@ -38,6 +38,13 @@ contract GnosisSafe
     //);
     bytes32 private constant SAFE_MSG_TYPEHASH = 0x60b3cbf8b4a223d68d641b3b6ddf9a298e7f33710cf3d3a9d1146b5a6150fbca;
 
+    event SafeSetup(
+        address indexed initiator,
+        address[] owners,
+        uint256 threshold,
+        address initializer,
+        address fallbackHandler
```

Comment:
`address paymentToken, uint256 payment, address payable paymentReceiver` should be there until we remove them

Reply:
I left them out as they do not have an impact on the internal state of the safe. And as they cost gas I would only include critical information.